oms-scomacs-solution/azuredeploy.json

```diff
@@ -0,0 +1,1504 @@
+                "Standalone",
+                "PerNode"
+            ],
+            "metadata": {
+                "description": "Service Tier: Free, Standalone, or PerNode"
+            }
+        }
+    },
+    "variables": {
+        "omsSolutions": {
+            "customSolution": {
+                "name": "SCOM ACS Dashboards",
+                "solutionName": "[concat('SCOMACSDashboards', '[', parameters('workspaceName'), ']')]",
+                "publisher": "Adin.Ermie@outlook.com",
+                "displayName": "SCOM ACS Dashboards",
+                "description": "This is a pre-built solution for Microsoft OMS (www.microsoft.com/oms).This solution was built from following Wei Ho Lim's 2 articles. This solution requires the OMS Security and Audit solution to be enabled first, as it leverages this collected data.",
+                "author": "Adin.Ermie@outlook.com"
+            }
+        },
+        "LogAnalyticsApiVersion": "2015-11-01-preview",
+        "SecurityAndAuditSolution": {
+            "Name": "[Concat('Security', '(', parameters('workspaceName'), ')')]",
+            "GalleryName": "Security"
+        }
+    },
+    "resources": [
+        {
+            "name": "[parameters('workspaceName')]",
+            "type": "Microsoft.OperationalInsights/workspaces",
+            "apiVersion": "[variables('LogAnalyticsApiVersion')]",
+            "location": "[parameters('workspaceRegionId')]",
+            "properties": {
+                "sku": {
+                    "name": "[parameters('pricingTier')]"
+                }
+            },
+            "resources": [
+                {   
+                    "apiVersion": "[variables('LogAnalyticsApiVersion')]",
+                    "location": "[parameters('workspaceRegionId')]",
+                    "name": "[variables('SecurityAndAuditSolution').Name]",
+                    "type": "Microsoft.OperationsManagement/solutions",
+                    "id": "[concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/', resourceGroup().name, '/providers/Microsoft.OperationsManagement/solutions/', variables('SecurityAndAuditSolution').Name)]",
+                    "dependsOn": [
+                        "[concat('Microsoft.OperationalInsights/workspaces/', parameters('workspaceName'))]"
+                    ],
+                    "properties": {
+                        "workspaceResourceId": "[resourceId('Microsoft.OperationalInsights/workspaces/', parameters('workspaceName'))]"
+                    },
+                    "plan": {
+                        "name": "[variables('SecurityAndAuditSolution').Name]",
+                        "publisher": "Microsoft",
+                        "product": "[Concat('OMSGallery/', variables('SecurityAndAuditSolution').GalleryName)]",
+                        "promotionCode": ""
+                    }
+                },
+                {
+                    "apiVersion": "[variables('LogAnalyticsApiVersion')]",
```

Comment:
type the apiVersion on the resources directly, instead of using the apiVersion in a variable. This is for best practices and tooling